test.js

```diff
@@ -35,6 +35,18 @@ test('.flagToScope(flag)', async () => {
   assert.equal(starryNight.flagToScope('whatever'), undefined)
 })
 
+test('.createStarryNight with options', () => {
+  assert.rejects(
+    async () => {
+      await createStarryNight(common, {
+        getOnigurumaUrlFs: () => new URL('file:///foo/baz/onig.wasm')
+      })
+    },
+    /no such file or directory, open '\/foo\/baz\/onig.wasm'/,
+    "should throw when file at given URL can't be found"
```

Comment:
```suggestion
    'should support `getOnigurumaUrlFs`'
```

Please adhere to the code style used in the project: running `npm install` and `npm test` should check and format these things and make sure everything works as expected.

Reply:
63eb1bd complies with the code style